### PR TITLE
Update to ALPHA-4

### PR DESCRIPTION
**Pull request by AppleDevelops**
- [x] This PR actually submits something - don't use this system as a messaging service!
- [x] This PR isn't duplicated - you can check if it is by scanning the small list - link is on the navigation bar for this repository.
- [x] This PR includes appropriate markdown for sections - e.g. code blocks for suggested code.
- [x] This PR description and comments in code is understandable - feel free to use your native language to write if you are not comfortable with English.
- [x] This PR has a single branch for itself in your fork - don't just commit to the master branch of your repository!
- [x] This PR has comments written in clear English - please don't write unreadable comments just for your eyes.

## Pull Request description
This updates to ALPHA-4 of PMMP, I can't tell if I missed anything so let me know.

## Changed Branch
Master

## Optional information
...

### CODE REVIEW

File: src/revivalpmmp/pureentities/PureEntities.php [hunk 1]
```diff
@@ -255,13 +255,14 @@ public static function create($type, Position $source, ...$args) {
...
         } else {
             $chunk = $source->getLevel()->getChunk($source->x >> 4, $source->z >> 4, true);
+			$level = $source->getLevel();
             if (!$chunk->isGenerated()) {
                 $chunk->setGenerated();
             }
             if (!$chunk->isPopulated()) {
                 $chunk->setPopulated();
             }
-            return Entity::createEntity($type, $chunk, $nbt, ...$args);
```
Comment (kniffo80) [inline src/revivalpmmp/pureentities/PureEntities.php]: The IF statement is now not needed anymore. I saw that Tesseract has been updated. So the IF clause can be removed and only Level should be used!

**Comment by kniffo80** [comment]
When PureEntities is changed again (spawning) to only use Level - this PR is ok.

File: src/revivalpmmp/pureentities/PureEntities.php (hunk 1 above)
Comment (AppleDevelops) [inline src/revivalpmmp/pureentities/PureEntities.php]: Gotcha

**Comment by NycuRO** [comment]
[Tesseract] [19:41:54] CRITICAL> TypeError: "Argument 1 passed to revivalpmmp\pureentities\tile\Spawner::__construct() must be an instance of pocketmine\level\Level, instance of pocketmine\level\format\Chunk given, called in phar:///root/fac/Tesseract_1.1.0_2017-02-24_06-15-02.phar/src/pocketmine/tile/Tile.php on line 106" (EXCEPTION) in "/phar_PureEntitiesX-master (2)_ZhoGVK2GJV7L5hu.phar/src/revivalpmmp/pureentities/tile/Spawner" at line 44

**Comment by kniffo80** [comment]
No time this weekend.

**Comment by kniffo80** [comment]
Changes have been applied in branch: 
https://github.com/RevivalPMMP/PureEntitiesX/tree/kniffo80-dev

Please use this atm. Development will continue. I'll close this here therefore.

**Pull request closed without merging**